templates/addons/s3/policy.yml

```diff
@@ -0,0 +1,23 @@
+{{.BucketResourceName}}AccessPolicy:
+  Type: AWS::IAM::ManagedPolicy
+  Properties:
+    Description: !Sub
+      - Grants CRUD access to the S3 bucket ${Bucket}
```

Comment:
What's the $ syntax?

Reply:
Subs in the value in the key/value pair on the line below for the key ${Bucket}

Reply:
This renders as 
![image](https://user-images.githubusercontent.com/25392995/83461206-f71af500-a41c-11ea-8f28-b5c7a85bdb84.png)